website/common/script/libs/getItemInfo.js

```diff
@@ -191,7 +192,7 @@ module.exports = function getItemInfo (user, type, item, officialPinnedItems, la
     case 'gear':
       // spread operator not available
       itemInfo = Object.assign(getDefaultGearProps(item, language), {
-        value: item.twoHanded ? 2 : 1,
+        value: item.value,
```

Comment:
`value` under `gear` needs to have the `item.twoHanded ? 2 : 1` because this shows the gem price, the values under `marketGear` is the one paid with gold